reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -16,6 +22,20 @@
   z-index: 2;
   background-color: white;
   color: #133a53;
+  
+}
+
+.darkmode table th {
+  color: #f2f2f2;
+}
+
+.darkmode .table td,
+.table th {
```

Comment:
What is the intention behind this block?

Reply:
The 2 selectors weren't needed. I changed it to `.table th {border-top: none;}`
This is needed to stop the scrolling table from peeking out above the column titles.

Reply:
.table th is too generic a selector - make it heatmap specific

Reply:
no longer using this block

Reply:
The block might be gone, but the .table th selector still exists

Reply:
removed